### PR TITLE
feat(unlock-app): Show information on key in a separate model and simplify the design

### PR DESCRIPTION
**Pull request by searchableguy**
# Description

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

https://user-images.githubusercontent.com/73341821/206578447-5e3ce98b-3136-4c86-b79c-f25772f7f3b6.mp4

CleanShot 2022-12-09 at 03 44 16@2x

### CODE REVIEW

**Comment by julien51** [comment]
hey, as discussed, the token Id, Expire On, and Price should actually *not* appear on the "front" but in a modal that opens when people click on a specific key. This will ensure they are all of the same size.

**Comment by searchableguy** [comment]
CleanShot 2022-12-09 at 19 26 25@2x

File: unlock-app/src/components/interface/keychain/Key.tsx [hunk 1]
```diff
@@ -453,11 +325,11 @@ function Key({ ownedKey, account, network }: Props) {
                         active={active}
                         onClick={(event) => {
                           event.preventDefault()
-                          setShowMetadata(true)
+                          setShowMoreInfo(true)
                         }}
                       >
                         <InfoIcon />
-                        View metadata
+                        Show membership info
```
Comment (julien51) [inline unlock-app/src/components/interface/keychain/Key.tsx:332]:
```suggestion
                        Show details
```

File: unlock-app/src/components/interface/keychain/KeyInfoDrawer.tsx [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+          )}
+        {keyPrice && keyPrice.amount !== '0.0' && (
+          <KeyItem label="Price">
+            {`${keyPrice.amount} ${keyPrice.symbol}`}
+          </KeyItem>
+        )}
+        {subscription && (
+          <KeyItem label="Renewals">
+            {subscription.possibleRenewals <= 0
+              ? 'Renewal will not happen due to low balance'
```
Comment (julien51) [inline unlock-app/src/components/interface/keychain/KeyInfoDrawer.tsx:211]: Can we replace with "your XXX is too low for renewals".

**Review by julien51 — APPROVED** [review]

**Pull request merged**